core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -133,7 +133,8 @@ trait EndToEndAuthorizationTest extends IntegrationTestHarness with SaslSetup {
   this.serverConfig.setProperty(KafkaConfig.AuthorizerClassNameProp, classOf[SimpleAclAuthorizer].getName)
   // Some needed configuration for brokers, producers, and consumers
   this.serverConfig.setProperty(KafkaConfig.OffsetsTopicPartitionsProp, "1")
-  this.serverConfig.setProperty(KafkaConfig.OffsetsTopicReplicationFactorProp, "1")
+  this.serverConfig.setProperty(KafkaConfig.OffsetsTopicReplicationFactorProp, "3")
+  this.serverConfig.setProperty(KafkaConfig.OffsetCommitRequiredAcksProp, "1")
```

Comment:
Hi @ijuma with the topic replication factor was 1, because the min.insync.replicas was 3, the offset commits (whichI understand are msgs sent with ack=-1) failed.
I first upped OffsetsTopicReplicationFactorProp to 3, but I still had failures due to timeouts I think, so I then set OffsetCommitRequiredAcksProp=1.

Are you suggesting to only keep OffsetCommitRequiredAcksProp="1" and not have
OffsetsTopicReplicationFactorProp="3" ?

thanks!